### ISSUE TITLE
File Uploader: Accessibility issues

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#4665]
File Uploader Accessibility issues

Attached screenshot
[File Uploader.docx](https://github.com/SAP/fundamental-ngx/files/5980063/File.Uploader.docx)

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue persists for the labels are not associated and not read by the screen reader.
As discussed will be opening ticket for the association label.
hence closing this ticket.

Thanks and Regards
Sanjay
